virt-lightning.org/etc/caddy/conf.d/virt-lightning.org.conf

```diff
@@ -30,8 +30,8 @@ virt-lightning.org {
     redir /images/centos-7/centos-7.qcow2 http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2
     redir /images/centos-6/centos-6.qcow2 https://cloud.centos.org/centos/6/images/CentOS-6-x86_64-GenericCloud.qcow2
     redir /images/centos-8/centos-8.qcow2 https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
-    redir /images/centos-8-stream/centos-8-stream.qcow2 https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-latest.x86_64.qcow2
-    redir /images/centos-9-stream/centos-9-stream.qcow2 https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-latest.x86_64.qcow2
```

Comment:
Could you actually keep these two lines to avoid breaking the users who rely on the current name?